larpandora/LArPandoraInterface/LArPandoraGeometry.h

```diff
@@ -300,6 +344,17 @@ namespace lar_pandora {
                                     const unsigned int cstat,
                                     const unsigned int tpc);
 
+    /**
+     *  @brief  Get daughter volume ID from a specified cryostat/tpc pair
+     *
+     *  @param  driftVolumeMap the output mapping between cryostat/tpc and drift volumes
+     *  @param  cstat the input cryostat unique ID
+     *  @param  tpc the input tpc unique ID
+     */
+    static unsigned int GetDaughterVolumeID(const LArDriftVolumeMap& driftVolumeMap,
+                                    const unsigned int cstat,
+                                    const unsigned int tpc);
```

Comment:
Compress to 1-2 lines and standard indentation here and elsewhere.

Reply:
Here also I mimicked the rest of the file (which I guess is the way larsoft formatted it)